packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts

```diff
@@ -27,23 +45,18 @@ export const HorizontalScrollTemplate: ViewTemplate<HorizontalScroll> = html`
             ${when(
                 x => x.view !== "mobile",
                 html<HorizontalScroll>`
-                    <div
-                        class="scroll scroll-prev"
-                        part="scroll-prev"
-                        ${ref("previousFlipper")}
-                    >
-                        <div class="scroll-action" @click="${x => x.scrollToPrevious()}">
-                            <slot name="previous-flipper"></slot>
-                        </div>
-                    </div>
                     <div
                         class="scroll scroll-next"
                         part="scroll-next"
                         ${ref("nextFlipper")}
                     >
-                        <div class="scroll-action" @click="${x => x.scrollToNext()}">
+                        <button
+                            class="scroll-action"
+                            @click="${x => x.scrollToNext()}"
+                            tabindex="-1"
+                        >
```

Comment:
Same note as above - I think we should target the feature branch on this and leverage the default slotted content API's. In that case, we can also slot the actual flippers themselves.